chapter_optimization/momentum_vn.md

```diff
@@ -123,15 +123,17 @@ See e.g., the discussion by :cite:`Sutskever.Martens.Dahl.ea.2013` for details.
 ### An Ill-conditioned Problem
 -->
 
-### *dịch tiêu đề phía trên*
+### Bài toán với Điều kiện Xấu
 
 <!--
 To get a better understanding of the geometric properties of the momentum method we revisit gradient descent, albeit with a significantly less pleasant objective function.
 Recall that in :numref:`sec_gd` we used $f(\mathbf{x}) = x_1^2 + 2 x_2^2$, i.e., a moderately distorted ellipsoid objective.
 We distort this function further by stretching it out in the $x_1$ direction via
 -->
 
-*dịch đoạn phía trên*
+Để hiểu hơn về các tính chất hình học của phương pháp động lượng, chúng ta hãy ôn lại thuật toán hạ gradient với hàm mục tiêu khó chịu hơn.
+Hãy nhớ lại trong :numref:`sec_gd` chúng ta sử dụng $f(\mathbf{x}) = x_1^2 + 2 x_2^2$ là hàm mục tiêu dạng elip méo.
```

Comment:
```suggestion
Hãy nhớ lại trong :numref:`sec_gd` chúng ta sử dụng $f(\mathbf{x}) = x_1^2 + 2 x_2^2$ là hàm mục tiêu dạng elip biến dạng.
```

Em thấy nó không giống bị méo lắm

Reply:
a cũng dịch vậy. @cuongvng phản hồi gì k

Reply:
Anh cũng nhớ ngày xửa xưa đọc trong sách nào đó họ cũng hay có từ méo. Nhưng a không chắc lắm, lúc dịch a cũng hơi phân vân :))

Reply:
ơ mà phương trình `x_1^2 + 2 x_2^2` là của elip thật, `distorted` chỗ nào nhỉ?

Reply:
@cuongvng cùng thắc mắc :))

Reply:
Vote bỏ 'distorted'.

Reply:
```suggestion
Hãy nhớ lại trong :numref:`sec_gd` chúng ta sử dụng $f(\mathbf{x}) = x_1^2 + 2 x_2^2$ là hàm mục tiêu dạng elip.
```

Reply:
Anh k giỏi toán lắm nhưng có thắc mắc là sao lại gọi `f(x) = ...` là phương trình nhỉ, nó là hàm số mà. Phương trình của elip là `ax1^2 + bx2^2 = 1`, ở đây vế phải là một hằng số, vế trái là tập hợp x1, x2 thoả mãn PT này, từ đó tạo nên 1 elip (chuẩn k méo).  Nhưng f(x) chỉ đơn thuần là vế trái, thay đổi khi x1, x2 thay đổi. Từ đó dẫn đến các đường cung elip khác nhau (không hiểu có phải vậy mà họ gọi là méo/biến dạng liên tục k).
 
![image](https://user-images.githubusercontent.com/30741785/86510341-cec25400-be29-11ea-8b7e-791257822778.png)
https://d2l.ai/chapter_optimization/momentum.html

Reply:
à, có vẻ a phỏng đoán hơi hơi đúng, maybe ý tác là với hàm elip như vầy, tác giả làm méo hàm elip bằng cách kéo dãn nó ra: `We distort this function further by stretching it out`  
![image](https://user-images.githubusercontent.com/30741785/86510495-11386080-be2b-11ea-8df7-70770b84758e.png)

Reply:
Em thấy dù kéo thế nào nó vẫn theo phương trình ellipse...

Reply:
>à, có vẻ a phỏng đoán hơi hơi đúng, maybe ý tác là với hàm elip như vầy, tác giả làm méo hàm elip bằng cách kéo dãn nó ra: We distort this function further by stretching it out

kéo dãn elip cũng vẫn chỉ là elip thôi mà, với cả có từ `further` thì em nghĩ `distort` là kiểu đang là hình tròn `x1^2 + x2^2` kéo ra thành elip bằng cách thêm 2 hệ số vào.

Reply:
nên nếu vẫn muốn giữ `distorted` thì mn xem thế này ổn không?
```suggestion
Hãy nhớ lại trong :numref:`sec_gd` chúng ta đã làm méo hàm mục tiêu thành dạng elip $f(\mathbf{x}) = x_1^2 + 2 x_2^2$.
```

Reply:
> Em thấy dù kéo thế nào nó vẫn theo phương trình ellipse...
Anh đồng ý là với một giá trị cụ thể của hàm f(x), e.g., `x_1^2 + 2 x_2^2 = 1` , thì tập x1, x2 thoả mãn tạo thành 1 hình elip. Nhưng khi ta cực tiểu hàm f(x) này về 0.5 chẳng hạn, thì ta thu được một hình elip khác (mập hơn hoặc gầy hơn :))). 

Ý của Cường khá gần với với ý của anh. Là hình tròn làm méo thì thành hình elip A, hình elip A khi làm méo thì sẽ thành hình elip B (kiểu 1 người bình thường, sau khi học đc 1 vài thứ linh tinh thì nhân cách méo mó dù người đó vẫn là con người). Méo ở đây là tương quan giữa hai cái hình elip A và elip B chứ k ám chỉ là `elip méo`.

a nghĩ ý tác giả ở đây, là khởi đầu bằng một hình elip (vì ban đầu ta biết tính đc 1 giá trị của thể của f(x)). và cái từ distorted ở đây là ám chỉ nó sẽ bị làm `méo` / chuyển thành các hình khác.

Reply:
`làm méo ... thành dạng elip` thì không hẳn đúng, nhưng cũng có thể là tác giả đang so sánh với hình elip x^2 + y^2 :)) a nghĩ vậy thôi chứ a cũng hơi mơ hồ :'(

Reply:
>khởi đầu bằng một hình elip (vì ban đầu ta biết tính đc 1 giá trị của thể của f(x)). và cái từ distorted ở đây là ám chỉ nó sẽ bị làm méo / chuyển thành các hình khác.

vậy ổn nhất em nghĩ nên bỏ `distorted` đi, vì kể cả khi ý tác giả là vậy thì từ này cũng không quan trọng lắm trong context.
```suggestion
Hãy nhớ lại trong :numref:`sec_gd` chúng ta sử dụng $f(\mathbf{x}) = x_1^2 + 2 x_2^2$ là hàm mục tiêu dạng elip.
```